### ISSUE TITLE
Load/Unload model 

### ISSUE THREAD

**Issue by bangpc** [triton-inference-server/server#4976]
**Description**

**Triton Information**
I use triton image 22.08

**To Reproduce**
I use fisheye model, I try to send the following config to server via `load_model` function
```json 
{
    "name": "fisheye",              
    "platform": "onnxruntime_onnx", 
    "default_model_filename": "model.onnx",
    "input": [
        {
            "name": "input",                
            "data_type": "FP32",            
            "dims": [1, 3, 608, 608]    
        }
    ],
    "output": [
        {
            "name": "output",               
            "data_type": "FP32",            
            "dims": [1, 22743, 6]       
        }
    ],
    "version_policy": {"specific": {"versions": [1]}}
}
```
As discuss on [this issue](https://github.com/triton-inference-server/server/issues/4416#issuecomment-1260285216), @dyastremsky told me that I send wrong config via `load_model`. I try with the config above but no luck, and I also do not know what is correct form of config when using `load_model`

**Expected behavior**
Exact model version loaded when call `load_model` function

**Comment by bangpc** [comment]
Problem solved,
I send the following config to `load_model` function and it work
```json 
{
    "version_policy": {"specific": {"versions": [1]}}
}
```

**Comment by dyastremsky** [comment]
Thanks for solving this yourself. For the future, we have sample model configurations [here](https://github.com/triton-inference-server/server/blob/main/docs/user_guide/model_configuration.md#minimal-model-configuration). I suspect the extra brackets (the ones around the config) are resulting in the config being considered invalid.

**Comment by krishung5** [comment]
@bangpc This issue might have something to do with the `data_type` as well. It should be `TYPE_FP32` instead of `FP32`. I guess that's why it works after you removed the input/output part. After the fix in 22.09, there should be some error message in the server log if there is anything wrong with the config.
